src/components/ErrorPage403.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './NotFound.less';
+import NotFoundSVG from '../img/not-found.svg';
+import { components } from '../params';
+
+class ErrorPage403 extends React.Component {
+  render() {
+    const supportEmail = components.login?.email || 'support@datacommons.io';
+    return (
+      <div className='error-placeholder__error-msg_no_header_footer '>
```

Comment:
This class name doesn't exists in the stylesheets that this component has imported

Reply:
Thanks. I missed this, I added a new class to make sure the css hack gets applied, but later when I changed the iframe to redirect, I eliminated the css class, but forgot to eliminate this implementation.